x/stakeibc/keeper/icqcallbacks_validator_exchange_rate_test.go

```diff
@@ -43,8 +46,10 @@ func (s *KeeperTestSuite) SetupValidatorICQCallback() ValidatorICQCallbackTestCa
 	// We don't actually need a transfer channel for this test, but we do need to have IBC support for timeouts
 	s.CreateTransferChannel(HostChainId)
 
-	valAddress := "valoper1"
-	valIndexQueried := 0 // index in the validators array
+	// These must be valid addresses, otherwise the bech decoding will fail
```

Comment:
Was the test previously failing?

Reply:
Previously we weren't catching the error case of if you couldn't decode the address. Once I changed the original function to catch that error case, this test failed (since "valoper1" is not a valid address)

Reply:
Nice catch!